template/java8/model/src/test/java/RequestTest.java

```diff
@@ -39,14 +42,14 @@ public void testEmptyRequestParameterGetSet()
 	@Test
 	public void testRequestRawGetSet()
 	{
-		Request request = new Request(null,null,"testRaw=testRawValue", null);
+		Request request = new Request(null,"testRaw=testRawValue", null, null);
 		assertEquals("testRaw=testRawValue", request.getQueryRaw());
 	}
 
 	@Test
 	public void testGetPath()
 	{
-		Request request = new Request(null,null, null, "/test/path");
+		Request request = new Request(null, null, "/test/path", new ByteArrayInputStream(new byte[0]));
```

Comment:
Unless the final parameter is optional, then this looks like it might be a breaking change?

Reply:
Yes it is. I will make this parameter optional.